reversi/src/ConsoleUI.hs

```diff
@@ -0,0 +1,69 @@
+module ConsoleUI where
+
+import GameLogic
+import Board
+
+import Data.Char (chr, ord, isSpace)
+import Data.List (dropWhileEnd)
+
+import Text.Read (readMaybe)
+
+renderBoard :: Board -> String
+renderBoard board =
```

Comment:
It doesn't render nicely for sizes above 10*10

Reply:
[Fixed](https://github.com/kajigor/fp-2024-cub-fall/pull/169/commits/88cf9cda0cfc6f0da023dbe56d0d9dc829299946#diff-f32d6b9f729a45b3180bfd68586b7b117cbc5197474fd6d9658d9b6917a4add4)